### ISSUE TITLE
Visual C++ instructions incorrect

### ISSUE THREAD

**Issue by dmwysocki61** [michaelrsweet/mxml#222]
Instructions to open solution file in Visual Studio are incorrect. Visual studio does not find the projects even though they are in the same folder as the solution file. Editing a solution file should not be required as there is no documentation anywhere that clearly describes what changes are needed. 

Step to copy lib file and header file is incomplete / incorrect. There is no include folder in VS. The lib file present does not match the instructions.

Bottom line, the sample programs do not complile. Errors too numerous to list.

**Comment by michaelrsweet** [comment]
Strange, they work for me.

What version of Visual Studio do you have?

**Comment by dmwysocki61** [comment]
Thanks for the reply. Not sure what I'm doing wrong. I have VS 2017 community version. 
I figured out how to get the test program to compile with harbour so I may give up on the C++ version.
I'll give it a try again and send additional details as to the error messages. Dave Wysocki Diversified Tech Systems,Inc. 954-328-3328

**Comment by michaelrsweet** [comment]
Please post additional information when you have it. I will verify that it works on my system with the current Visual Studio tools before I post Mini-XML 2.12.
